apps/portal/src/app/views/notifications/notifications.component.ts

```diff
@@ -24,6 +24,14 @@ export class NotificationsComponent implements OnInit {
   deliveryStatusMap = {
     [DeliveryStatus.PENDING]: { value: 'Pending', style: 'pending' },
     [DeliveryStatus.IN_PROGRESS]: { value: 'In Progress', style: 'in-progress' },
+    [DeliveryStatus.AWAITING_CONFIRMATION]: {
+      value: 'Awaiting Confirmation',
+      style: 'awaiting-confirmation',
+    },
+    [DeliveryStatus.QUEUED_CONFIRMATION]: {
+      value: 'Queued confirmation',
```

Comment:
Can maintain consistency in value, capitalizing the first letter of each word like others.

Reply:
Updated